src/com/actelion/research/gwt/minimal/MoleculeQueryFeatures.java

```diff
@@ -0,0 +1,83 @@
+package com.actelion.research.gwt.minimal;
+
+import com.actelion.research.chem.StereoMolecule;
+import com.actelion.research.chem.Molecule;
+import com.google.gwt.core.client.JavaScriptObject;
+
+class MoleculeQueryFeatures {
+  public static JavaScriptObject getAtomQueryFeatures(StereoMolecule oclMolecule, int atom) {
+    PlainJSObject moleculeQueryFeatures = PlainJSObject.create();
+
+    moleculeQueryFeatures.setPropertyBoolean("aromatic",
+        (oclMolecule.getAtomQueryFeatures(atom) & Molecule.cAtomQFAromatic) > 0);
```

Comment:
Wouldn't it be significantly faster to call `getAtomQueryFeatures(atom)` only once and put the value in a local variable?

Reply:
Indeed good idea! The method `getAtomQueryFeatures` is super small so probably small impact but the code will look any much better and we will have the possibility to optimise the part that is 'int' and not long. Thanks.
```
public long getAtomQueryFeatures(int atom) {
return mAtomQueryFeatures[atom];
}
```